src/coreclr/scripts/genEventPipe.py

```diff
@@ -178,9 +178,19 @@ def generateWriteEventBody(template, providerName, eventName):
 
     bool success = true;
 """ % (template.estimated_size, template.estimated_size)
-
     fnSig = template.signature
     pack_list = []
+
+    # Write out replacement for any instasnces of NULL for UnicodeString type input.
+    # ETW translates these to "NONE", so we are doing the same for EventPipe.
```

Comment:
The comment says NONE, but the code below uses NULL. Is it right?

Reply:
oops - yes it should be "NULL" : )